apps/vscode/vscode.talon

```diff
@@ -11,41 +11,8 @@ app: Visual Studio Code
 app: Code.exe
 -
 
-# editing
-code refactor all: user.ide_refactor()
-code refactor: user.ide_refactor_in_line()
-code rename: user.ide_refactor_rename()
-code follow: user.ide_follow()
-code comment: user.ide_toggle_comment()
-code suggest: user.ide_smart()
-code arguments: user.ide_parameter_hints()
-code done: user.ide_done()
-
-# navigating
-go back: user.ide_go_back()
-go forward: user.ide_go_forward()
-
-# selection
-cursor up: user.ide_up_cursor()
-cursor down: user.ide_down_cursor()
+# general
 
-# debugging
-step over: user.ide_step_over()
-step into: user.ide_step_into()
-step continue: user.ide_continue()
+open containing folder | show in explorer | show in finder: user.ide_reveal_in_file_manager()
```

Comment:
I don't see a reason not to include this in the generic IDE.talon, to be honest. That way, if you forget which OS you're on, it'll still work.